File: events/ready.ts
import { startSpan } from "@sentry/node";
import { Client } from "discord.js";

export default async function ready(client: Client) {
    console.info(`Logged in as ${client.user?.username}!`);
    await client.guilds.fetch();
    client.user!.setPresence({
        activities: [{ name: `${client.guilds.cache.size} Servers`, type: 3 }],
    });
}

File: events/ready.py
from loguru import logger as log
from sentry_sdk import start_transaction
from discord import Cog, Bot, Activity, ActivityType


class Ready(Cog):

    def __init__(self, bot: Bot) -> None:
        self.bot = bot

    @Cog.listener()
    async def on_ready(self) -> None:
        with start_transaction(name="on_ready"):
            log.info("logged in as {}!", self.bot.user.name)
            activity = Activity(
                name=f"{len(self.bot.guilds)} Servers", type=ActivityType.watching
            )
            await self.bot.change_presence(activity=activity)
